src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetail.component.tsx

```diff
@@ -15,8 +15,7 @@
  */
 
 import React, { useEffect, useRef, useState, useMemo } from 'react'
-import { NavLink, Redirect, Route, Switch } from 'react-router-dom'
-import { useParams, useRouteMatch, useLocation } from 'react-router'
+import { NavLink, Redirect, Route, Switch, useParams, useRouteMatch, useLocation } from 'react-router-dom'
```

Comment:
Check for formatting

Reply:
printWidth is 120 characters long. Since this line is only 107 characters long it won't be formatted into separate lines.